CMakeLists.txt

```diff
@@ -104,6 +105,13 @@ if (WERROR)
   message (STATUS "Warnings as errors enabled (-Werror), disable with -DWERROR=off")
 endif (WERROR)
 
+# Remove Warnings for "unused return"  ?
+if (WUNUSED)
+  list (APPEND C_CXX_WARNING_FLAGS -Wno-unused-result)
+   message (STATUS "Warnings for unused-result enabled, disable with -DWUNUSED=on")
```

Comment:
The text: "...disable with -DWUNUSED=on"... Shouldn't that be "OFF", instead of "on"?

Reply:
... checks to see if it's ON.  But here, if WUNUSED == ON, you tell them to set it to ON again to disable...  Hence the confusing logic...

What am I missing?

Reply:
i'll make the message more explicit